core/rpc/client/gateway/jsonrpc/client.go

```diff
@@ -0,0 +1,188 @@
+package jsonrpc
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"reflect"
+	"strconv"
+	"sync/atomic"
+
+	rpcClient "github.com/kwilteam/kwil-db/core/rpc/client"
+	"github.com/kwilteam/kwil-db/core/rpc/client/gateway"
+	jsonrpc "github.com/kwilteam/kwil-db/core/rpc/json"
+)
+
+const (
```

Comment:
Maybe should move to `core/rpc/json` ?

Reply:
maybe not, admin and function methods are in `core/rpc/json` because it's also used in JSON rpc service server.